### PR TITLE
Upload scout variants for Balsamic

### PR DESCRIPTION
**Pull request by moonso**
This PR fixes a problem that no variants where loaded when uploading balsamic cases to scout.

Also cg will now print the load config in yaml format instead of json when printing to screen.

### How to prepare for test
- [x] ssh to relevant server hasta
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh upload-scout-variants`

### How to test load config is printed in yml format
- [x] `cg upload scout actualgecko --print`

### Expected test outcome
- [x] check that the load config is printed in yaml format
- [x] Take a screenshot and attach or copy/paste the output.

Screen Shot 2021-02-02 at 17 14 40

### How to test variants are uploaded
- [x] `cg upload scout actualgecko --re-upload`

### Expected test outcome
- [x] check that the variants was uploaded to scout
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by
- [x] tests executed by
- [x] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: cg/meta/upload/scout/balsamic_config_builder.py [hunk 1]
```diff
@@ -48,14 +48,17 @@ def build_config_sample(self, db_sample: models.FamilySample) -> ScoutBalsamicIn
         )
         if analysis_type == "tgs":
             analysis_type = "panel"
+        if analysis_type == "wgs":
+            raise SyntaxError("Not allowed to upload WGS cancer cases")
```
Comment (Mropat) [inline cg/meta/upload/scout/balsamic_config_builder.py:54]: Do we maybe wanna we use a custom CgError for this?

File: cg/cli/upload/scout.py [hunk 2]
```diff
@@ -52,18 +54,26 @@ def create_scout_load_config(context, case_id: str, print_console: bool, re_uplo
     LOG.info("Fetching family object")
     case_obj: Family = status_api.family(case_id)
     LOG.info("Create load config")
-    scout_load_config: ScoutLoadConfig = scout_upload_api.generate_config(case_obj.analyses[0])
+    try:
+        scout_load_config: ScoutLoadConfig = scout_upload_api.generate_config(case_obj.analyses[0])
```
Comment (Mropat) [inline cg/cli/upload/scout.py:58]: why not have a check?
if not case_obj.analyses:
    LOG.bla()
    raise click.Abort

?

File: cg/meta/upload/scout/balsamic_config_builder.py (hunk 1 above)
Comment (moonso) [reply cg/meta/upload/scout/balsamic_config_builder.py:54]: I thought it was ok since it will be removed anyday

**Review by Mropat — APPROVED** [review]
:+1:

File: cg/cli/upload/scout.py (hunk 2 above)
Comment (moonso) [reply cg/cli/upload/scout.py:58]: oki

**Comment by Mropat** [comment]
Deployed on prod
```
Building wheels for collected packages: cg
  Running setup.py bdist_wheel for cg ... done
  Stored in directory: /home/maryia.ropart/tmp/pip-ephem-wheel-cache-ym0exwjl/wheels/5c/23/37/e3635140fad58f81760c6db5705a64683daf19230a7bb26dda
Successfully built cg
Installing collected packages: cg
  Found existing installation: cg 18.11.0
    Uninstalling cg-18.11.0:
      Successfully uninstalled cg-18.11.0
Successfully installed cg-18.11.1
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/stage/nipt/cases/lovedkitten
done.
Log deploy... done.
Running --version on each Clinical-Genomics developed tool to verify that all requirements are met:
argcomplete 1.12.1 has requirement importlib-metadata<3,>=0.23, but you have importlib-metadata 0.8.
cg, version 18.11.1
housekeeper, version 3.2.0
demux, version 5.5.1
cglims, version 1.3.4

```

**Pull request merged**